nbs/core.ipynb

```diff
@@ -94,7 +94,8 @@
     "    MLPMultivariate, iTransformer,\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        AutoMLP(h=12, config=config_f, num_samples=2, backend='optuna')
</pre><p>Please also set the sampler so that the trials are the same across runs, i.e.</p>

<pre class="ql-syntax" spellcheck="false">AutoMLP(h=12, config=config_f, num_samples=2, backend='optuna', search_alg=optuna.samplers.TPESampler(seed=0))
</pre><p>Also please add another model with ray so that we test both backends and include exogenous in the tuning parameters, one constant and one being tuned, e.g.</p>

<pre class="ql-syntax" spellcheck="false">config = {
    'hist_exog_list': ['a'],
    'futr_exog_list': tune.choice([['b'], ['c'], ['b', 'c']])
}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
ah yes it seems extra logic was needed when we are tuning the hyperparameter
```
for exog in exogs:
  if isinstance(exog, str):
      hist_exog.append(exog)
  else:
      hist_exog.extend(exog)
```